src/main/app/claims/models/claim.ts

```diff
@@ -38,6 +38,7 @@ export class Claim {
   id: number
   claimantId: string
   externalId: string
+  state: string
```

Comment:
would be nice to have this of type enum `ClaimState`

Reply:
I haven't set it as enum as that would imply that claimstore/frontend enum have to be in sync or there would be a prod issue. Since it would introduce an extra layer of coupling and it would be a kind of major decision, I decided to leave it as a string and maybe introduce a tech debt / huddle discussion about that

Reply:
I hear your fear, we already have few enums but those may not be changing at all. We have some tendency to get new states in future. Let's keep it `string` for the time being.